### PR TITLE
fix: dev docker container

### PR DESCRIPTION
**Pull request by jleach**
There was an issue (for me) running the devContainer because I'm on an ARM64 machine and it was expecting an AMD64 architecture. This PR fixes that issue by supporting both architectures.

### CODE REVIEW

File: aries-mobile-tests/config.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+        "sauce:options": {
+          "appiumVersion": "latest",
+          "fullReset": true,
+          "sauceLabsImageInjectionEnabled": true,
+          "sessionCreationRetry": "3"
+        }
+      }
+    ],
+    "platformName": "",
+    "app": ""
```
Comment (nodlesh) [inline aries-mobile-tests/config.json]: Why are platformName and app added here? If the configs need it, the sl_ios_config and sl_android_config will need them. When the tests are run from the manage script these two config files are updated with the data form the command line options and then copied to the config.json. So adding things to the config.json in the repo won't do much.

Comment (nodlesh) [reply aries-mobile-tests/config.json]: Yes, these extra platformName and app need to be removed.

**Review by nodlesh — CHANGES_REQUESTED** [review]
Ran it, works perfectly for me. Just need the change in the config.json file.

File: aries-mobile-tests/config.json (hunk 1 above)
Comment (jleach) [reply aries-mobile-tests/config.json]: Not sure how they got there but will fix.

Comment (jleach) [reply aries-mobile-tests/config.json]: Done.

**Review by nodlesh — APPROVED** [review]
Looks good.

**Pull request merged**